Api/PaymentApiClient.php

```diff
@@ -48,11 +48,12 @@ public function getNewWebPayment(string $storeId, array $parameters = []): int
 
     /**
      * @param string $storeId
-     * @param int $storeKeeperId
+     * @param int|string $storeKeeperId
      * @param array $paymentIds
-     * @retrun void
+     * @return void
+     * @throws \Exception
      */
-    public function attachPaymentIdsToOrder(string $storeId, int $storeKeeperId, array $paymentIds = []): void
+    public function attachPaymentIdsToOrder(string $storeId, int|string $storeKeeperId, array $paymentIds = []): void
```

Comment:
```suggestion
    public function attachPaymentIdsToOrder(string $storeId, int $storeKeeperId, array $paymentIds = []): void
```
This one is strange because it's never string